mindbender/tools/projectmanager/app.py

```diff
@@ -51,8 +51,8 @@ def __init__(self, parent=None):
 
         body = QtWidgets.QSplitter()
         body.setContentsMargins(0, 0, 0, 0)
-        body.setSizePolicy(QtGui.QSizePolicy.Expanding,
-                           QtGui.QSizePolicy.Expanding)
+        body.setSizePolicy(QtWidgets.QSizePolicy.Expanding,
```

Comment:
A little worrysome, sounds like the vendored Qt.py might have gotten affected by another local Qt.py?